frontend/src/lib/derived/tokens-list-user.derived.ts

```
@@ -47,15 +55,39 @@ const addActions = (userTokenData: UserTokenData): UserTokenData => ({
         ],
 });
 
+const addHref = ({
+  userTokenData,
+  authData,
+}: {
+  userTokenData: UserTokenData;
+  authData: AuthStoreData;
+}): UserTokenData => ({
+  ...userTokenData,
+  rowHref: isNullish(authData.identity)
```

This derived store is not called when there is no identity, but still, we need to check this.

---

Since the plan is for `rowHref` to be optional only temporarily, how do you plan to satisfy the type system here when it becomes mandatory?
Maybe it makes sense to do that already?

---

As I mentioned [here](https://github.com/dfinity/nns-dapp/pull/4000#issuecomment-1849623996), I'm not sure yet it should be mandatory.

Here I would throw an error or force a logout probably. It's an edge case that shouldn't happen.